Tests/AK/TestStringUtils.cpp

```diff
@@ -319,6 +319,13 @@ TEST_CASE(convert_to_uint_from_octal)
     EXPECT_EQ(actual.value(), 0177777u);
 }
 
+TEST_CASE(convert_to_floating_point)
+{
+    auto number_string = "  123.45  "sv;
+    auto maybe_number = AK::StringUtils::convert_to_floating_point<float>(number_string, TrimWhitespace::Yes);
+    EXPECT_EQ(maybe_number, 123.45f);
```

Comment:
Thanks for the test! Looks good, except let's use `EXPECT_APPROXIMATE` here. That will relax the test to ensure we are just within some epsilon of the expected value, as floating point representation isn't perfectly exact.